src/testing/euler_gaussian_bump.cpp

```diff
@@ -80,7 +97,7 @@ ::run_test () const
         unsigned int n_grids = n_grids_input;
         if (poly_degree <= 1) n_grids = n_grids_input;
 
-        std::vector<double> entropy_error(n_grids);
+        std::vector<double> Error(n_grids);
```

Comment:
avoid using capital letters for variable names

Reply:
Fixed it.